README.md

```diff
@@ -1,151 +1,149 @@
+For some packages, notably CompCert and VST (the Princeton tool-chain for verification of C code), exist various variants.
 
-You can change the selection of packages any time later by issuing `opam install` commands, e.g.
-```
-opam install coq-compcert.3.8
-opam install coq-vst.2.7.1
-```
+By default the 64 bit variant of CompCert and the 64 bit variant of VST are installed.
 
-There is also a 32 bit variant of CompCert and VST, which can be installed by running these commands:
+You can install the 32 bit variants in addition any time later by issuing `opam install` commands, e.g.
 ```
-opam install coq-compcert-32.3.8
-opam install coq-vst-32.2.7.1
+opam install coq-compcert-32.3.9
+opam install coq-vst-32.2.8
 ```
-or by adjusting [`coq_platform_packages.sh`](/coq_platform_packages.sh) accordingly.
 Please note that since both variants can be installed in parallel, only one, the 64 bit variant, is immediately available to Coq
-without -Q and -R options. If you want to work with the 32 bit variants, please use these options in your Coq project:
+without -Q and -R options.
+If you want to work with the 32 bit variants, please use these options in your Coq project:
 ```
 -Q $(coqc -where)/../coq-variant/compcert32/compcert compcert
 -Q $(coqc -where)/../coq-variant/VST32/VST VST
 ```
 
-### Installation of other packages
+**Important note:** CompCert is **not** free / open source software, but may be used for research and evaluation purposes.
+Please clarify the license at [CompCert License](https://github.com/AbsInt/CompCert/blob/master/LICENSE).
+
+## Installation of other packages
 
 - On Windows open a shell with `C:\<your_coq_platform_cygwin_path>\cygwin.bat`.
 - On Linux or macOS open a shell in the usual way.
-- Run these commands:
+- Run the command `opam switch` which will show the list of available switches:
+    ```
+    #   switch                                 compiler                    description
+        __coq-platform.2021.09.0~8.12          ocaml-base-compiler.4.10.0  __coq-platform.2021.09.0~8.12
+        __coq-platform.2021.09.0~8.13          ocaml-base-compiler.4.10.0  __coq-platform.2021.09.0~8.13
+        __coq-platform.2021.09.0~8.13~2021.02  ocaml-base-compiler.4.10.0  __coq-platform.2021.09.0~8.13~2021.02
+    ->  __coq-platform.2021.09.0~8.14+beta1    ocaml-base-compiler.4.10.0  __coq-platform.2021.09.0~8.14+beta1
+        _coq-platform_.2021.02.1               ocaml-base-compiler.4.07.1  _coq-platform_.2021.02.1
     ```
-    opam switch __coq-platform.2021.09.0.8.13
+- Choose the switch you want to change with this command (example):
+    ```
+    opam switch __coq-platform.2021.09.0~8.13
     eval $(opam env)
     ```
-    (note: the switch name might vary if you choose a different version of Coq - please use `opam switch` to see a list of switch names)
 - Install additional packages with `opam install "package"`
 - You can find packages with `opam list --all | grep "some keyword"`
```

Comment:
Maybe adding a link from there to https://coq.inria.fr/opam-using.html#coq-packages would make sense.